independent-publisher-2/functions.php

```diff
@@ -6,138 +6,154 @@
  */
 
 if ( ! function_exists( 'independent_publisher_2_setup' ) ) :
-/**
```

Comment:
It looks like some phpcs + phpcbf changes are getting checked into this PR as well.